src/java/org/apache/cassandra/index/sai/plan/CountFetchedTransformation.java

```diff
@@ -67,14 +67,14 @@ protected Row applyToRow(Row row)
         if (row.hasLiveData(nowInSec, false))
             queryContext.addRowsFetched(1);
         else
-            queryContext.addTombstonesFetched(1);
+            queryContext.addRowTombstonesFetched(1);
         return row;
     }
 
     @Override
     protected RangeTombstoneMarker applyToMarker(RangeTombstoneMarker marker)
     {
-        queryContext.addTombstonesFetched(1);
+        queryContext.addRowTombstonesFetched(1);
```

Comment:
If we are adding checkpoint everywhere, shouldn't we add one in this method too?